projects/angular/package.json

```diff
@@ -20,6 +20,6 @@
   "peerDependencies": {
     "@angular/common": "^12.0.0",
     "@angular/core": "^12.0.0",
-    "@cds/core": "^5.0.0"
+    "@cds/core": ">5.3.0"
```

Comment:
should we be explicit about v5 and v6 here?

Reply:
My first change was `>5.3.0 || ^6.0.0' and then I got thinking about it and since we started with core the entire idea behind it was - no public breaking changes. So following that idea it won't be an issue.

Reply:
Wait, what do you mean "no public breaking changes"? The design token change in v6 was a breaking change, we may have breaking changes in v7

Reply:
I think we should be more explicit here.